### PR TITLE
Dev v0.7.1

### PR DESCRIPTION
**Pull request by sarahmarshy**

### CODE REVIEW

File: project_generator/init_yaml.py [hunk 1]
```diff
@@ -41,9 +41,9 @@ def _scan(section, root, directory, extensions, is_path):
 def _generate_file(filename,root,directory,data):
         logging.debug('Generating yaml file')
         if os.path.isfile(os.path.join(directory, filename)):
-            print("Project file already exists")
+            print("Project file " +filename+  " already exists")
             while True:
-                answer = input('Should I overwrite it? (Y/n)')
+                answer = raw_input('Should I overwrite it? (Y/n)')
```
Comment (sarahmarshy) [inline project_generator/init_yaml.py]: input() function runs eval() on user input, IE, previous way was trying to resolve y,N or whatever you type to an expression

File: project_generator/project.py [hunk 2]
```diff
@@ -214,7 +213,7 @@ def __init__(self, name, project_dicts, pgen_workspace):
     def _fill_project_defaults(self):
 
         self.project = {
-            'name': self.name,          # project name
+            'name': '',          # project name
```
Comment (0xc0170) [inline project_generator/project.py]: Why is it emptY? :)

**Comment by 0xc0170** [comment]
Tests are red, extensions of generated projects suddenly are not correct

**Comment by 0xc0170** [comment]
@sarahmarshy Can you please fix the tests ?

**Comment by sarahmarshy** [comment]
Bad merge, I think :(. Tests failing in areas I did not edit.

**Comment by 0xc0170** [comment]
OK, cherry pick  your commits and add them to the latest master - that might help

**Pull request closed without merging**